lib/register/index.js

```diff
@@ -0,0 +1,33 @@
+import check from '../check';
+import { code, message } from '../consts';
+
+/**
+ * Self register to window load
+ * Throw the error globally
+ */
+export default function register() {
+    function asynchronous() {
+
+        // Throw on next tick
+        setTimeout(() => {
+            const files = check();
+
+            if (files.length) {
+                const error = new Error([ message, ...files.map(({ src }) => src) ].join('\n'));
+                error.code = code;
+                error.files = files;
+
+                error.toJSON = () => ({
```

Comment:
This is interesting.. Can you elaborate on why you did it?

Reply:
Upon [JSON stringification ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#toJSON_behavior) this error it should include the error code and exclude the list of node elements

Reply:
👍🏻